components/mindstorms-ev3/src/main/eventflow/com/example/ev3/Demo.sbapp

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<modify version="10.6.1_0a0fdf7f8f3d4f25851d53e0e55c97ce2ece3d22">
+    <add>
+        <annotations>
+            <annotation name="hygienic"/>
+        </annotations>
+        <type-metadata>
+            <param name="type" value="module"/>
+            <param name="fully-qualified-name" value="com.example.ev3.Demo"/>

```

Comment:
Same as above, `com.example` should be `com.tibco.ep.community.components` everywhere.